docs/standard/garbage-collection/latency.md

```diff
@@ -7,42 +7,50 @@ helpviewer_keywords:
+To reclaim objects, the garbage collector (GC) must stop all the executing threads in an application. *Latency* refers to the time that the garbage collector intrudes in your application. In some situations, such as when an application retrieves data or displays content, a full garbage collection can occur at a critical time and impede performance. You can adjust the intrusiveness of the garbage collector by setting the <xref:System.Runtime.GCSettings.LatencyMode%2A?displayProperty=nameWithType> property to one of the <xref:System.Runtime.GCLatencyMode?displayProperty=nameWithType> values.
 
-- <xref:System.Runtime.GCLatencyMode.LowLatency> suppresses generation 2 collections and performs only generation 0 and 1 collections. It can be used only for short periods of time. Over longer periods, if the system is under memory pressure, the garbage collector will trigger a collection, which can briefly pause the application and disrupt a time-critical operation. This setting is available only for workstation garbage collection.
+## Low latency settings
 
-- <xref:System.Runtime.GCLatencyMode.SustainedLowLatency> suppresses foreground generation 2 collections and performs only generation 0, 1, and background generation 2 collections. It can be used for longer periods of time, and is available for both workstation and server garbage collection. This setting cannot be used if [concurrent garbage collection](../../../docs/framework/configure-apps/file-schema/runtime/gcconcurrent-element.md) is disabled.
+During low latency periods, the garbage collector is more conservative and less intrusive in reclaiming objects.
 
- During low latency periods, generation 2 collections are suppressed unless the following occurs:
+The <xref:System.Runtime.GCLatencyMode?displayProperty=nameWithType> enumeration provides two low latency settings:
+
+- [GCLatencyMode.LowLatency](xref:System.Runtime.GCLatencyMode.LowLatency) suppresses generation 2 collections and performs only generation 0 and 1 collections. It can be used only for short periods of time. Over longer periods, if the system is under memory pressure, the garbage collector will trigger a collection, which can briefly pause the application and disrupt a time-critical operation. This setting is available only for workstation garbage collection.
+
+- [GCLatencyMode.SustainedLowLatency](xref:System.Runtime.GCLatencyMode.SustainedLowLatency) suppresses foreground generation 2 collections and performs only generation 0, 1, and background generation 2 collections. It can be used for longer periods of time, and is available for both workstation and server garbage collection. This setting cannot be used if background garbage collection is disabled by the .NET Framework [\<gcConcurrent> configuration element](../../../docs/framework/configure-apps/file-schema/runtime/gcconcurrent-element.md).
```

Comment:
Given that this is a .NET standard article, is the reference to .NET Framework appropriate here, i.e., are we saying that LowLatency works cross-platform but SustainedLowLatency only works on Framework?